src/ansys/fluent/visualization/contour.py

```diff
@@ -20,18 +36,41 @@ def _error_check(self, solver):
                 )
 
     def __init__(self, field: str, surfaces: List[str], solver: Optional = None):
+        """Create contour using field name and surfaces list.
+
+        Parameters
+        ----------
+        field : str
+            field name
+        surfaces : List[str]
+            surfaces name list
+        solver : Optional
+            solver session
+        """
         self.field = field
         self.surfaces = surfaces
         if solver:
             self.solver = solver
             self._error_check(self.solver)
 
     def _get_contour_name(self):
+        """Get random contour name."""
         import time
 
         return f"Contour_{time.time()}"
 
     def draw(self, solver, target):
+        """Creates a Graphics(solver) or solver based contour object.
+
+        Parameters
+        ----------
+        solver : solver session.
+        target : either Graphics(solver) or solver session.
```

Comment:
Format should be parameter name followed by a space, a colon, a space, and then the data type. The next line is the brief description, indented four spaces.

Reply:
Done. Thank you.